static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -0,0 +1,382 @@
+// @flow
+import formInlineLabels from 'lib/formInlineLabels';
+import bean from 'bean';
+import fastdom from 'fastdom';
+import $ from 'lib/$';
+import config from 'lib/config';
+import { getUserAgent } from 'lib/detect';
+import fetch from 'lib/fetch';
+import mediator from 'lib/mediator';
+import template from 'lodash/utilities/template';
+import { logError } from 'lib/robust';
+import { trackNonClickInteraction } from 'common/modules/analytics/google';
+import { inlineSvg } from 'common/views/svgs';
+import successHtml from 'raw-loader!common/views/email/submissionResponse.html';
+import closeHtml from 'raw-loader!common/views/ui/close-button.html';
```

Comment:
Bothe templates are only used by this module and are not longer than 5 lines. Could you convert them to inlined template literals?